src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -451,6 +451,43 @@ function entropy_projection!(cache, u, mesh::DGMultiMesh, equations, dg::DGMulti
   return nothing
 end
 
+# TODO: REFACTOR. redundant with entropy_projection!
```

Comment:
This part is useless in this PR. Maybe better to remove for now? But I have a question about the design choice: Seems like 1D Gauss collocation is dispatched with `DGMulti` instead of `DGMultiFluxDiff,` is there any reason behind this? Seems more intuitive to use `DGMultiFLuxDiff{<:GaussSBP}` for 1D Gauss to me

Reply:
Can you link where it's dispatched with `DGMulti`? It should be `DGMultiFluxDiff{<:GaussSBP}`...

Reply:
Oh, maybe I don't know how to use DGMulti properly. So my question is: if I want to use 1D Gauss SBP, I will construct `dg = DGMulti(polydeg = ..., element_type = Line(), approx_type = Polynomial(), surface_integral = WeakForm(), volume_integral = FluxDifferencing())`, instead of using `GaussSBP()` in the approximation type, is it correct? and GaussSBP only supports Quad and Hex type from what I understand. Is there a particular reason to do use Polynomial() in 1D? Or I missed something? Thanks!